packages/kit/src/exports/vite/graph_analysis/index.js

```diff
@@ -255,11 +267,12 @@ export class ViteImportGraph {
  * @param {(id: string) => import('rollup').ModuleInfo | null} node_getter
  * @param {import('rollup').ModuleInfo} node
  * @param {string} lib_dir
+ * @param {IllegalModuleGuardOptions} [options]
```

Comment:
I believe the rollup check is working and it's only the vite check that's broken, so I would remove this and have it only affect `dev` and not `build`